src/parser.c

```diff
@@ -559,3 +559,23 @@ size_t pos_to_str_with_limit(struct pos_array *pos, uint8_t *str, size_t limit)
     }
     return len;
 }
+
+int pos_is_zero(struct pos_array *pos)
```

Comment:
Agree, I've thought about that.

The error messages are completely different in some cases:
```
Redis:
> select
(error) ERR wrong number of arguments for 'select' command

Corvus:
> select
(error) ERR Proxy fail to forward command
```

I'm afraid it's too much work to get them exactly the same in this PR.

And if nothing goes wrong, do we have to do this?